doc/api/http2.md

```diff
@@ -3150,13 +3150,15 @@ will result in a [`TypeError`][] being thrown.
 added: v8.4.0
 -->
 * `headers` {HTTP/2 Headers Object} An object describing the headers
-* `callback` {Function}
+* `callback` {Function} Callback that is called once
+`http2stream.pushStream()` is finished, or when the stream is closed.
+  * `err` {Error}
+  * `stream` {ServerHttp2Stream} The newly-created `ServerHttp2Stream` object.
 
 Call [`http2stream.pushStream()`][] with the given headers, and wraps the
-given newly created [`Http2Stream`] on `Http2ServerResponse`.
-
-The callback will be called with an error with code `ERR_HTTP2_INVALID_STREAM`
-if the stream is closed.
+given [`Http2Stream`] on a newly created `Http2ServerResponse` as the callback
+parameter when callback is successful. And for the closed stream, The callback
```

Comment:
Grammar/spelling nit:
```
For the closed stream, the callback
```

Reply:
Any spelling wrong or……?

Reply:
OK. Thanks. I'll also collect other suggestions and do modifications together :)